debian/control

```diff
@@ -48,6 +48,15 @@ Description: nymea integration plugin for ZigBee Spec compliant lighting devices
  which adhere to the ZigBee Cluster Library specification for lighting devices.
 
 
+Package: nymea-plugin-zigbee-gewiss
```

Comment:
maybe also add it to the `nymea-plugins-zigbee` meta package since we kept it for zigbee plugins

Reply:
done